frontend/src/app/components/ExperimentRowCard/index.scss

```diff
@@ -139,47 +118,72 @@
   }
 
   .tab {
-    border-bottom-right-radius: $large-border-radius;
-    border-bottom-width: 3px;
-    border-left-width: 0;
-    border-right-color: $white;
-    border-right-width: 3px;
-    border-style: solid;
-    border-top-left-radius: $small-border-radius;
-    border-top-width: 0;
+    border-style: solid;   
+    border-width: 1px;
+    border-color: rgb(58, 133, 36);
+    border-radius: $small-border-radius;
     display: inline-block;
     font-weight: 300;
     height: $grid-unit * 1.5;
-    left: -3px;
+    top: -5px;
+    left: -5px;
     line-height: $grid-unit * 1.5;
     outline: none;
     padding: 0 20px;
     position: absolute;
     text-align: center;
     text-decoration: none;
-    top: -3px;
     transition: background 150ms;
     white-space: nowrap;
     z-index: 99999;
   }
 
   .enabled-tab {
     background: $green-60;
-    border-bottom-color: $white;
     color: $white;
   }
 
+  .enabled-tab::before {
+    background-image: url('./img/check.svg');
+    content: '';
+    display: inline-block;
+    height: 14px;
+    width: 14px;
+    margin-right: 4px;
+  }
+
   .just-launched-tab {
-    background: $bright-blue;
-    border-bottom-color: $white;
+    background: $green-60;
     color: $white;
   }
 
+  .just-launched-tab::before {
+    background-image: url('./img/star.svg');
+    position: relative;
+    top: 2px;
+    content: '';
+    display: inline-block;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
   .just-updated-tab {
-    background: $bright-blue;
-    border-bottom-color: $white;
+    background: $green-60;
     color: $white;
   }
+
+  .just-updated-tab::before {
+    background-image: url('./img/star.svg');
+    position: relative;
+    top: 2px;
+    content: '';
+    display: inline-block;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
 }
 
 .experiment-icon-wrapper {
```

Comment:
Icon is a bit large here per spec. But this might actually be one of the conflicts with my PR, since I tweaked the size there too: https://github.com/mozilla/testpilot/pull/3038/files#diff-1a7df85b60eb52db9f0d6ddc67c725a2